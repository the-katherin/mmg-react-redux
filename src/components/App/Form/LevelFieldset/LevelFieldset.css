.level-fieldset__radios {
    display: grid;
    grid-auto-flow: column;
    grid-column-gap: 10px;
    justify-content: center;

}