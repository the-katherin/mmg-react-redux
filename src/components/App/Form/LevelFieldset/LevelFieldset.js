import React from 'react';

const LevelFieldset = () => (
    <fieldset className='level-fieldset' name='level'>

        <legend>Choose level</legend>

        <div className="level-fieldset__radios">

            <div>

                <input type="radio" name='level' id='easy' value="3" required />
                <label htmlFor="easy">Easy</label>

            </div>

            <div>

                <input type="radio" name='level' id='medium' value="6" />
                <label htmlFor="medium">Medium</label>

            </div>

            <div>

                <input type="radio" name='level' id='professional' value="12" />
                <label htmlFor="professional">Professional</label>
            </div>

        </div>
    </fieldset>
);

export default LevelFieldset;