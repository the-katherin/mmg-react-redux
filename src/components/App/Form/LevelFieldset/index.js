import './LevelFieldset.css';

export { default } from './LevelFieldset';