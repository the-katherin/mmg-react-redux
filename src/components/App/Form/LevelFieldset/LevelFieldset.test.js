import React from "react";
import LevelFieldset from "./LevelFieldset";

describe("LevelFieldset", () => {
    test("match snapshot", () => {
        const wrapper = shallow(<LevelFieldset />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});