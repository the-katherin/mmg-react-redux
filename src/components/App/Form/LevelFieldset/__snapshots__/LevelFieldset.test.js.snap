// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LevelFieldset match snapshot 1`] = `
<fieldset
  className="level-fieldset"
  name="level"
>
  <legend>
    Choose level
  </legend>
  <div
    className="level-fieldset__radios"
  >
    <div>
      <input
        id="easy"
        name="level"
        required={true}
        type="radio"
        value="3"
      />
      <label
        htmlFor="easy"
      >
        Easy
      </label>
    </div>
    <div>
      <input
        id="medium"
        name="level"
        type="radio"
        value="6"
      />
      <label
        htmlFor="medium"
      >
        Medium
      </label>
    </div>
    <div>
      <input
        id="professional"
        name="level"
        type="radio"
        value="12"
      />
      <label
        htmlFor="professional"
      >
        Professional
      </label>
    </div>
  </div>
</fieldset>
`;
